doc/index.rst

```diff
@@ -26,6 +26,16 @@ Table of Contents
    Software reference <_autodoc/qiskit>
    Release history <releases>
 
+Qiskit Tutorials
```

Comment:
I want to make the sections part of the main. You will see in the other pr that they come after install but before contribute.

Reply:
Sure, I'll move this when I respin the PR.